Main.cpp

```diff
@@ -2,8 +2,56 @@
 #include <iostream>
 #include <cstdlib>
 
+#include <regex>
+using namespace std;
+
 int main()
 {
+	//Part 1 of Assignment 1 BEGIN

```

Comment:
Question 1 asked for a standalone program, rather than something integrated into the unit-test program.  So you would do your work inside an `int main() { /* do the work*/}` in some `question1main.cpp`, with a separate project file or makefile that builds its own .exe.

As far as the overall approach... well, I was really just expecting basic usage of `std::cin >> foo` and `std::cout << foo`.

But dang - you went and taught yourself std::regex instead!  I'm impressed.  Points off for missing the intent, but some  consolation points back for skipping ahead.